crates/ide-completion/src/item.rs

```diff
@@ -424,6 +426,8 @@ impl Builder {
             }
         } else if let Some(trait_name) = self.trait_name {
             label = SmolStr::from(format!("{label} (as {trait_name})"));
+        } else if let Some(doc_aliases) = self.doc_aliases {
+            label = SmolStr::from(format!("{label} (alias {doc_aliases})"));
```

Comment:
Here is something we should do differently. I think we should split the completions according to their aliases up, that is for each alias we copy the item but change the label and lookup to `thing (alias <the alias>)` + the final one without its alias. This should be done in `to_proto.rs` similar to how we handle the `CompletionItem::ref_match` field.

Reply:
Well if setting lookup with multiple aliases somehow works out then sure, but I feel like it won't. Do try it out thought since it would simplify things 👍